ui/ts/pages/stores.ts

```diff
@@ -138,7 +141,24 @@ module AdminViews {
      * StorePage show the details of a single node.
      */
     export module StorePage {
+      import NavigationBar = Components.NavigationBar;
+
       class Controller {
+        private static defaultTargets: NavigationBar.Target[] = [
+          {
+            title: "Overview",
+            route: ""
+          },
+          {
+            title: "Graphs",
+            route: "graph"
+          }
+        ];
+
+        private static isActive: (targ: NavigationBar.Target) => boolean = (t: NavigationBar.Target) => {
```

Comment:
shouldn't this isactive be part of the navigationBar targets instead of the navigation bar itself? Each one should have its own isActive based on the route?